cypress/integration/dib-travel/personal-settings/profile/profile.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { Profile } from '../../../../models';
+
+describe('Personal settings - Profile page', () => {
+  let profileDetails: Profile;
+
+  before(() => {
+    cy.fixture('profile/profile').then((profile) => {
+      profileDetails = profile;
+    });
+  });
+
+  beforeEach(() => {
```

Comment:
Your general use `beforeEach` should be something like:
```
cy.login();
cy.visit('/parent-feature/sub-feature');
```

And testing only sidebar link (and confirming loaded content) should be a separate single test.